src/components/Modeler.vue

```diff
@@ -515,8 +515,12 @@ export default {
     },
     removeUnsupportedElementAttributes(definition) {
       const unsupportedElements = ['documentation', 'extensionElements'];
+
       unsupportedElements.filter(name => definition.get(name))
         .forEach(name => definition.set(name, null));
```

Comment:
It seems that the `documentation` attribute has the "isMany" property set to `true`, and so _bpmn-moddle_ expects it to have a default value of `[]` instead of `null`. I've noticed that when I set the value to `undefined` instead, it correctly sets it to the expected default value.

Digging into the source code, this seems to be the case:
```js
if (isUndefined(value)) {
  // unset the property, if the specified value is undefined;
  // delete from $attrs (for extensions) or the target itself
  if (property) {
    delete target[propertyName];
  } else {
    delete target.$attrs[name];
  }
}
// ...
function isUndefined(val) {
  return typeof val === 'undefined';
}
```

I've made this change locally and it seems to fix the problem, without having to modify `definition.$descriptor.properties`.